.github/workflows/ios-build-test-fabric.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         id: install_pods
         continue-on-error: true
         working-directory: ${{ env.WORKING_DIRECTORY }}/ios
-        run: RCT_NEW_ARCH_ENABLED=1 pod install
+        run: pod install
```

Comment:
IMO we should keep this explicit, especially that we've had this already in place.

The same for `RCT_NEW_ARCH_ENABLED=0`

Reply:
From what I checked Podfile value overrides command anyway